source/Tectia/index.rst

```diff
@@ -0,0 +1,589 @@
+
+.. code:: shell
+
+   if ( -d "/opt/tectia/bin" ) ; then
+   setenv PATH "$PATH:/opt/tectia/bin"
+   endif
+
+   if ( -d "/opt/tectia/man" ) ; then
+   setenv MANPATH "$MANPATH:/opt/tectia/man"
+   endif
+
+
+6. Request a Tectia license.
+The Tectia software you just installed has a 45 day evaluation license, and works for all RDHPCS logons. To request an extended license, email ONE help request to the help desk of the system you use the most. Please use the subject **Tectia license request**.
+
+7. Install the Tectia license. 
+Complete this step before your 45 day evaluation license expires. Once you have received your Tectia "stc64.dat" license file via the help system, create the proper directory for it and move the file to the directory.
+
+.. code:: shell
+
+   cd <download directory>
+   mkdir /etc/ssh2/licenses/
+   mv stc64.dat /etc/ssh2/licenses/
+
+**Configuration**
+Tectia stores its configuration in ${HOME}/.ssh2, as an xml file called ssh-broker-config.xml. It is recommended to use the graphical configuration tool, ssh-tectia-configuration.
+
+1. Launch the configuration client (ssh-tectia-configuration).
+
+.. figure:: /images/rheltectia1.png
+
+2. In the Default Connection item, set a default user name.
+
+.. figure:: /images/rheltectia2.png
+
+3. Enable X11 Forwarding
+
+.. figure:: /images/rheltectia3.png
+
+4. Add a PKCS 11 library under the “Key Providers” item and click on the “Add” button.
+
+.. figure:: /images/rheltectia4.png
+
+* Add the full path to the Coolkey library. It should be "/usr/lib64/pkcs11/libcoolkeypk11.so". 
+
+.. figure:: /images/rheltectia5.png
+
+* Check to make sure this is the correct location.
+* Confirm that the PKCS 11 key providers contains the Coolkey library.
+
+5. Under "Connection Profiles, add a new connection profile.
+
+.. figure:: /images/rheltectia6.png
+
+* Set a profile name, for example “jet”.
+* Set the full hostname, for exmaple “bastion-jet.boulder.rdhpcs.noaa.gov”.
+* Apply the changes and then click OK.
+
+Once the Tectia Client has been configured, you can connect to any of the following CAC bastions.
+
+.. code:: shell
+
+   Boulder, CO
+
+   bastion-jet.boulder.rdhpcs.noaa.gov
+   bastion-theia.boulder.rdhpcs.noaa.gov
+   bastion-gaea.boulder.rdhpcs.noaa.gov
+
+   Princeton, NJ
+
+   bastion-jet.princeton.rdhpcs.noaa.gov
+   bastion-theia.princeton.rdhpcs.noaa.gov
+   bastion-gaea.princeton.rdhpcs.noaa.gov
+
+**Using the Tectia SSH Client**
+
+Once Tectia has been configured and the binary directory has been added to your path. You can ssh into to Jet using your CAC. The Tetica ssh command is “sshg3”.
+
+1. In a terminal window type “sshg3 jet” where “jet” is the name of the connection profile created under step 5 of the configuration.
+2. You will be prompted to save and accept the key for this bastion. The RFC4716 fingerprint is "c9:06:04:cf:e5:d3:07:e5:8e:3f:f2:c1:a6:22:fb:d8" for bastion-jet.boulder.rdhpcs.noaa.gov. You need to type “save”.
```

Comment:
.... accept a key for the bastion.    Remove the specific fingerprint, I'm sure it's stale.

Reply:
done